src/hmem_cuda.c

```diff
@@ -760,6 +760,12 @@ bool cuda_is_gdrcopy_enabled(void)
 	return hmem_cuda_use_gdrcopy;
 }
 
+int cuda_flush_rdma_operations(void) {
+	enum cudaFlushGPUDirectRDMAWritesTarget target = cudaFlushGPUDirectRDMAWritesTargetCurrentDevice;
+	enum cudaFlushGPUDirectRDMAWritesScope scope = cudaFlushGPUDirectRDMAWritesToOwner;
+	return cudaDeviceFlushGPUDirectRDMAWrites(target, scope);
+}
+
 #else
 
```

Comment:
Since you declare this new function for both HAVE_CUDA and not HAVE_CUDA, please add a stub function (return `-FI_ENOSYS`) for  `cuda_flush_rdma_operations` in the `#else` block as well .

Reply:
fixed.